data/docs/metrics-management/k8s-deployment-override.mdx

```diff
@@ -1,15 +1,17 @@
 ---
 date: 2024-10-28
-title: Configure k8s-infra otelDeployment to collect metrics from receivers
-id: k8s-deployment-override
+title: k8s-infra otelDeployment and otelAgent configuration
+id: k8s-infra-otel-config
```

Comment:
Was this doc shared with any user @srikanthccv ?
This change will change the URL.